### ISSUE TITLE
Inconsistent casing of PowerShell language when running `func templates list -l powershell`

### ISSUE THREAD

**Issue by Swimburger** [Azure/azure-functions-core-tools#2891]
This is a minuscule issue. When running `func templates list -l powershell` the output is:
```bash
PowerShell Templates:
  Azure Blob Storage trigger
  Azure Cosmos DB trigger
  Azure Event Grid trigger
  Azure Event Hub trigger
  HTTP trigger
  IoT Hub (Event Hub)
  Azure Queue Storage trigger
  SendGrid
  Azure Service Bus Queue trigger
  Azure Service Bus Topic trigger
  SignalR negotiate HTTP trigger
  Timer trigger

Powershell Templates:
  RabbitMQ trigger
```

The RabbitMQ trigger should be listed with the rest of the PowerShell templates under the correctly capitalized PowerShell.

**Comment by anthonychu** [comment]
Thanks for reporting this. I believe this is also causing 2 "powershell" entries to appear when you run `func new`.

**Comment by michaelpeng36** [comment]
Resolved by #2922 and #2925
